broker/src/main/resources/management-schema.xml

```diff
@@ -44,7 +44,9 @@
     <field name="receiverPartitionId" id="0" type="uint16"/>
     <field name="valueType" id="1" type="uint8"/>
     <field name="intent" id="2" type="uint8"/>
-    <data name="command" id="3" type="blob"/>
+    <field name="recordKey" id="3" type="uint64" presence="optional"/>
+
+    <data name="command" id="32" type="blob"/>
```

Comment:
:question: Why 32?

Reply:
:shrug: I chose something that is far away from the regular fields but tbh. I'm not sure how `id` is used, if it's a breaking change to change the value etc.

Reply:
It's fixed once used, but it doesn't matter much in terms of ordering. It's just you can never reuse the ID for compatibility. It's fine if there are gaps, or if ID 8 goes after ID 4 :shrug:

Reply:
Okay, that makes sense. :+1:
I'll leave it at 32 then.

Reply:
Why not :smile: Just to clarify, we usually just use the next one, and if we remove a field, we'd just never reuse the previous one (so there'd be a gap). We don't need to "reserve" space or anything. But I think you got it :smile: